### PR TITLE
Fix broken/dead external links 1st lot

### PR DESCRIPTION
**Pull request by javierlinked**
Folders:

- S00-intro
- S01-fundamentals

### CODE REVIEW

File: docs/S00-intro/L4-technical-requirements/index.md [hunk 1]
```diff
@@ -90,7 +91,7 @@ Even if JavaScript is not your first software language, it's really important yo
 *   [Markdown All in One](https://marketplace.visualstudio.com/items?itemName=yzhang.markdown-all-in-one){target=_blank} All you need to write Markdown
 *   [GitHub Linker](https://marketplace.visualstudio.com/items?itemName=gimenete.github-linker){target=_blank} Create links to fragments of code in GitHub
 
-We strongly recommend using VSCode in this course, particularly if you're new to coding, due to its extensive plugins. Other popular editors include [Atom,](https://atom.io/){target=_blank} [Sublime Text,](https://www.sublimetext.com/){target=_blank} and [IDEA](https://www.jetbrains.com/idea/){target=_blank} (written originally for Java but now supports many languages).
```
Comment (dinadeljanin) [inline docs/S00-intro/L4-technical-requirements/index.md:93]: I'm still so upset about Atom being sunsetted. RIP 🪦.

File: docs/S01-fundamentals/M1-cryptography/L2-pub-key-crypto-additional/index.md [hunk 2]
```diff
@@ -32,7 +32,7 @@
...
-* [Article: Keeping Secrets Secret (BBC)](https://www.rigb.org/christmaslectures08/html/activities/keeping-secrets-secret.pdf#page=5){target=_blank} This is a valuable resource explaining, in simple visual terms, the modular arithmetic underpinning the security of public key cryptography, hashing (which we'll learn about next) and any other one-way or "trapdoor" functions. It illustrates how you cannot break a private key's encryption with brute-force but can easily validate it if you have the accompanying public key.
+* [Article: Keeping Secrets Secret (BBC)](https://web.archive.org/web/20200924112725/https://www.rigb.org/christmaslectures08/html/activities/keeping-secrets-secret.pdf){target=_blank} This is a valuable resource explaining, in simple visual terms, the modular arithmetic underpinning the security of public key cryptography, hashing (which we'll learn about next) and any other one-way or "trapdoor" functions. It illustrates how you cannot break a private key's encryption with brute-force but can easily validate it if you have the accompanying public key.
```
Comment (dinadeljanin) [inline docs/S01-fundamentals/M1-cryptography/L2-pub-key-crypto-additional/index.md:35]: Great catch on this one!

**Review by dinadeljanin — APPROVED** [review]

File: docs/S00-intro/L4-technical-requirements/index.md (hunk 1 above)
Comment (javierlinked) [reply docs/S00-intro/L4-technical-requirements/index.md:93]: Me too! But now website is live again. When I checked atom.io was not even working. Abandonware is also cool.

File: docs/S01-fundamentals/M1-cryptography/L2-pub-key-crypto-additional/index.md (hunk 2 above)
Comment (javierlinked) [reply docs/S01-fundamentals/M1-cryptography/L2-pub-key-crypto-additional/index.md:35]: That article was good, I needed to rescue it.

**Pull request merged**